### DISCUSSION TITLE
大学図書館で使用できますか？

### DISCUSSION THREAD

**Discussion by m-uchigasaki** [next-l/enju_leaf#1558] [Q&A] [answered]
できそうですか？

**Comment by nabeta** [comment, marked as answer]
別スレッドにもありますが、現状ではNACSIS-CAT/ILL対応の見通しがないため、別途CATクライアントの併用が必要になります。
http://mba-web.co.jp/products/nii/milagro/milagro-ill.html

詳しい方のヘルプが必要です。
